### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by bspitzbart**
This pull request contains an expanded README for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,87 @@
...
-- PIL
-
+- pandas
 
 ## Installation
+Preliminaries:
+Login to Bridges via ssh using a Unix or Mac command line terminal.  Login is available to Bridges directly or through the XSEDE portal. Please see the <a href="https://portal.xsede.org/psc-bridges">Bridges User's Guide</a>.  
+
+For Windows Users:  
+Many tools are available for ssh access to Bridges.  Please see <a href="https://ubuntu.com/tutorials/tutorial-ubuntu-on-windows#1-overview">Ubuntu</a>, <a href="https://mobaxterm.mobatek.net/">MobaXterm</a>, or <a href="https://www.chiark.greenend.org.uk/~sgtatham/putty/">PuTTY</a>
```
Comment (iparask) [inline README.md]:
```suggestion
Many tools are available for ssh access to Bridges.  Please see [Ubuntu](https://ubuntu.com/tutorials/tutorial-ubuntu-on-windows#1-overview), [MobaXterm](https://mobaxterm.mobatek.net) or [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/)
```

File: README.md [hunk 2]
```diff
@@ -2,47 +2,87 @@
...
 
 ### PSC Bridges
-From source:
+Once you have logged into Bridges, you can follow one of two methods for installing iceberg-seals.
+
+Method #1 (Recommended):
```
Comment (iparask) [inline README.md]:
```suggestion
#### Method 1 (Recommended):  
```

File: README.md [hunk 3]
```diff
@@ -2,47 +2,87 @@
...
+$ virtualenv seals_env             # create a virtual environment to isolate your work from the default system.
+$ source seals_env/bin/activate    # activate your environment. Notice the command line prompt changes to show your environment on the next line.
+[seals_env] $ pwd
+/pylon5/group/username/Seals
+[seals_env] $ export PYTHONPATH=<path>/seals_env/lib/python3.5/site-packages # set a system variable to point python to your specific code. (Replace <path> with the results of pwd command above.
+[seals_env] $ pip install iceberg_seals.search # pip is a python tool to extract the requested software (iceberg_seals.search in this case) from a repository. (this may take several minutes).
 ```
 
-From PyPi:
+Method #2 (Installing from source; recommended for developers only):
```
Comment (iparask) [inline README.md]:
```suggestion
#### Method 2 (Installing from source; recommended for developers only): 
```

File: README.md [hunk 4]
```diff
@@ -2,47 +2,87 @@
...
-- pandas
 - geopandas
-- numpy
-- PIL
-
+- pandas
 
 ## Installation
+Preliminaries:
+Login to Bridges via ssh using a Unix or Mac command line terminal.  Login is available to Bridges directly or through the XSEDE portal. Please see the <a href="https://portal.xsede.org/psc-bridges">Bridges User's Guide</a>.
```
Comment (iparask) [inline README.md]:
```suggestion
Login to Bridges via ssh using a Unix or Mac command line terminal.  Login is available to Bridges directly or through the XSEDE portal. Please see the [Bridges User's Guide](https://portal.xsede.org/psc-bridges).  
```

File: README.md [hunk 5]
```diff
@@ -2,47 +2,87 @@
...
+[iceberg_seals] $ deactivate    # exit your virtual environment.
+$ interact -p GPU-small            # request a compute node (this may take a minute or two or more).
+$ cd $SCRATCH/Seals             # make sure you are in the same directory where everything was set up before.
+$ module load cuda                 # load parallel computing architecture, as before.
+$ module load python3              # load correct python version, as before.
+$ source seals_env/bin/activate # activate your environment, no need to create a new environment because the Seals tools are installed and isolated here.
+[iceberg_seals] $ iceberg_seals.detect --help  # this will display a help screen of available usage and parameters.
+```
+## Prediction
+- Download a pre-trained model at: https://github.com/iceberg-project/Seals/tree/feature/README/models/Heatmap-Cnt/UnetCntWRN/UnetCntWRN_ts-vanilla.tar
```
Comment (iparask) [inline README.md]: We may have to change this path when it is merged.

**Review by iparask — CHANGES_REQUESTED** [review]
Hello @bspitzbart , I made some markdown suggestions. The README has more information than before definitely.

It might be good to say somewhere that the package is tested on P100 GPUs. We use Bridges, but that does not exclude any other resource that offers the same GPUs.

In the installation, we can also say that the instruction are specific to Bridges and if someone wants to use another resource they have to make sure that cuda and python3 are present. This will allow them to skip the `module load` instructions which are specific to Bridges.

Also, can I ask you to remove `.README.md.un~`, `.setup.py.swp` and `README.md~`?

Thank you,
Giannis

File: README.md (hunk 5 above)
Comment (bspitzbart) [reply README.md]: Changed to master since the model is already there.

File: README.md [hunk 6]
```diff
@@ -2,47 +2,90 @@
...
-- pandas
 - geopandas
-- numpy
-- PIL
-
+- pandas
 
 ## Installation
+Preliminaries:  
+These instructions are specific to XSEDE/bridges but other resources can be used if cuda, python3, and a gpu processor are present, in which case 'module load' instructions can be skipped, which are specific to bridges.
```
Comment (iparask) [inline README.md]:
```suggestion
These instructions are specific to XSEDE Bridges but other resources can be used if cuda, python3, and a NVIDIA P100 GPU are present, in which case 'module load' instructions can be skipped, which are specific to Bridges.  
```

**Review by iparask — APPROVED** [review]
This is better. I have a very small correction to make things a bit more clear.

**Pull request merged**